tests/decorators/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -54,11 +54,13 @@ class OrderLine {
     this.price = price;
   }
 
-  @computed get total() {
+  @computed
+  get total() {
```

Comment:
This is now inline with flow and other decorator usages. Only for properties there should be a special case if I'm not mistaken. #1817 

I tried making class methods with one decorator inline, but there are more tests that want it split on multiple lines, so I went with this snapshot change.